src/helics/shared_api_library/MessageFederate.h

```diff
@@ -421,6 +421,27 @@ HELICS_EXPORT const char* helicsEndpointGetInfo(HelicsEndpoint end);
  */
 HELICS_EXPORT void helicsEndpointSetInfo(HelicsEndpoint endpoint, const char* info, HelicsError* err);
```

Comment:
The docstring for this on line 414 should be updated to say `endpoint` instead of `filter`